src/plugins/jqplot.donutRenderer.js

```diff
@@ -450,7 +454,10 @@
                 labelelem.css({left: x, top: y});
             }
         }
-               
+        if (this.totalLabel) {
+            var totalLabel = $('<div class="jqplot-data-label" style="position:absolute">' + this._totalAmount + '</div>').insertAfter(plot.eventCanvas._elem);
```

Comment:
I don't wanted to use other styling than the already existing one. Inline 'style="position:absolute' is already used for the existing labels in e.g. donutRenderer and pieRenderer. Maybe it would be a good idea to streamline this for all affected labels in all renderers. My intention was to reuse the existing styling logic.
However, if you want I could introduce a new class just for the central label?